<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tieba_space_style</class>
 <widget class="QMainWindow" name="tieba_space_style">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>778</width>
    <height>613</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>tieba_space_style</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>50</y>
      <width>561</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>330</y>
      <width>561</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="switch_button">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>300</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>转换</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clear_button">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>300</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>清除</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
